src/Grpc.Net.Client/Balancer/Internal/SocketConnectivitySubchannelTransport.cs

```diff
@@ -64,20 +65,23 @@ internal class SocketConnectivitySubchannelTransport : ISubchannelTransport, IDi
     internal Socket? _initialSocket;
     private BalancerAddress? _initialSocketAddress;
     private List<ReadOnlyMemory<byte>>? _initialSocketData;
+    private DateTime? _initialSocketCreatedTime;
     private bool _disposed;
     private BalancerAddress? _currentAddress;
 
     public SocketConnectivitySubchannelTransport(
         Subchannel subchannel,
         TimeSpan socketPingInterval,
         TimeSpan? connectTimeout,
+        TimeSpan connectionIdleTimeout,
```

Comment:
nit naming... "idle timeout" suggests "if idle for X, kill it" - but this actually only applies during connect; maybe `connectIdleTimeout`?

Reply:
"connect" in this context is the action of connecting the socket.

I renamed it to `socketIdleTimeout`. It's more accurate to what is idling.